### DISCUSSION TITLE
Action button on result page

### DISCUSSION THREAD

**Discussion by PixelAlarm** [photobooth-app/photobooth-app#378] [Ideas]
Hey, I went through the closed issues and didn't see a discussion on this topic.

What do you think about displaying the action buttons on the result page.
Maybe there is some non-intrusive way to add them (e. g. stacked unter der qr-code).

Often the users want to take several pictures in a row and have to close the result page first and then click on the action again.
This could safe one step :)

**Comment by mgineer85** [comment]
In v5 I just changed the layout of the detailed view a bit and moved the buttons from the top to the bottom so all action buttons are always in the same location and compared to before layout they are bigger:

![image](https://github.com/user-attachments/assets/7d712ed6-2a7c-4418-af30-95029d057f3c)

I think it could still be added somehow, but might be too much then.

**Reply by find-usdk** [reply]
It looks really good. I'm looking forward to this update being released. 

**Keep up the good work**

**Comment by PixelAlarm** [comment]
Nice. This seems to be less fiddly, especially on smaller screens.

And how about the buttons for taking a new photo / collage. Will it be possible to display them on the result page? (That’s what I meant)

**Reply by mgineer85** [reply]
Yes would be no problem to display them there also.
I just see the space is now occupied and could confuse to have two sets of different action buttons there looking almost the same?

maybe show to the very right just one button repeating the last chosen action? Or display all actions but more compact?

**Reply by PixelAlarm** [reply]
Since the new Layout is now released, I thought a bit about this idea.

Option 1: 
For my setup it would be possible to add the action buttons next to the result screen options.
I mostly only use 2-3 actions and only allow delete (and sometimes printing) on the result screen.
So it would be only up to 5 buttons in a row on the bottom (picture, collage, print, delete, filter)

Option 2:
Another option would be to move the result screen options (share, delete, filter) to the right under the qr button.
The action buttons (picture, collage) could be displayed at the same spot as on the frontpage.
That way the two "sections" would be separated and hopefully won't confuse the users.

**Comment by find-usdk** [comment]
Could there be made a timer, so that after example 3 min. without interaction it returns to the front screen?

**Reply by mgineer85** [reply]
Yes, was discussed recently here https://github.com/photobooth-app/photobooth-app/issues/377
Not sure yet about the exact implementation, but seems not too complicated 🙃
